source/app.js

```diff
@@ -30,6 +30,8 @@ function getCurrentRouteName(navigationState: NavigationState): ?string {
 
 type Props = {}
 
+const navigationPersistenceKey = __DEV__ ? null : 'NavState'
```

Comment:
This seems out of scope for this PR to me—this fix could get pulled out into its own PR.

Reply:
Done (#2813).

I'm leaving it in here until it's merged, so that the tests in this PR actually pass.

Reply:
That has been merged so this may be removed.